util/entropy_buffer_generator.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+""" Entropy Buffer Generator
+
+Creates an array of random bytes and stores it in a file.
+Typical usage:
+
+./entropy_buffer_generator.py -n 100
+
+This will create a file 'entropy_buffer.npy' consisting of 100 bytes.
+"""
+import argparse
+import random as rnd
+
+import numpy as np
+
+
+def parse_args():
+    """Parses command-line arguments."""
+    parser = argparse.ArgumentParser(
+        description="""The Entropy Buffer generator file for testing.
```

Comment:
Maybe it's worth adding a more explicit note here to say the randomness is not cryptographic-strength and should only be used for tests?

Reply:
Good idea. I've changed the note to be more clear.